content/operate/rc/supported-regions.md

```diff
@@ -0,0 +1,140 @@
+| `eu-south-1`   | Milan, Italy        | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `eu-south-2`   | Spain               | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `eu-west-1`    | Ireland             | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `eu-west-2`    | London, UK          | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `eu-west-3`    | Paris, France       | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+### Asia Pacific
+
+| Region ID        | Location             | Redis Cloud Pro | Redis Cloud Essentials |
+|------------------|----------------------|-----------------|------------------------|
+| `ap-east-1`      | Hong Kong, China     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `ap-northeast-1` | Tokyo, Japan         | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `ap-northeast-2` | Seoul, South Korea   | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `ap-south-1`     | Mumbai, India        | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `ap-south-2`     | Hyderabad, India     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `ap-southeast-1` | Singapore            | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `ap-southeast-2` | Sydney, Australia    | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `ap-southeast-3` | Jakarta, Indonesia   | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `ap-southeast-4` | Melbourne, Australia | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+### Middle East and Africa
+
+| Region ID      | Location                | Redis Cloud Pro | Redis Cloud Essentials |
+|----------------|-------------------------|-----------------|------------------------|
+| `af-south-1`   | Cape Town, South Africa | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `il-central-1` | Tel Aviv, Israel        | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `me-central-1` | UAE                     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `me-south-1`   | Bahrain                 | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+## Google Cloud
+
+Redis Cloud supports databases on all of the following Google Cloud regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 
```

Comment:
Same comment as in AWS. "All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. " is not necessary IMO

Reply:
```suggestion
Redis Cloud supports databases in the following Google Cloud regions. 
```